static/app/views/monitors/components/cronsLandingPanel.tsx

```diff
@@ -126,6 +130,22 @@ export function CronsLandingPanel() {
   const location = useLocation();
   const platform = decodeScalar(location.query?.platform) ?? null;
   const guide = decodeScalar(location.query?.guide);
+  const {
+    selection: {projects: selectedProjects},
+  } = usePageFilters();
+
+  const {projects} = useProjects();
+  const projectSlug =
+    selectedProjects.length === 1 &&
+    projects.find(project => project.id === String(selectedProjects[0]))?.slug;
```

Comment:
I think there's a way to pass the list of IDs to the `useProjects` to ensure they're loaded.

I'm sure we must have his pattern somewhere else 🤔